metrics/report/README.md

```diff
@@ -0,0 +1,48 @@
+* [Kata Containers metrics report generator](#kata-containers-metrics-report-generator)
+   * [Data gathering](#data-gathering)
+   * [Report generation](#report-generation)
+
+# Kata Containers metrics report generator
+
+The files within this directory can be used to generate a 'metrics report'
+for Kata Containers.
+
+The primary workflow consists of two stages:
+
+1) Run the provided report metrics data gathering scripts on the system(s) you wish
+to analyse.
+2) Run the provided report generation script to analyse the data and generate a
+report file.
+
+## Data gathering
+
+Data gathering is provided by the `grabdata.sh` script. When run, this script will
```

Comment:
Lines 19-21 suggested rewrite to stay active: ...When run, this script executes a set of tests from the `tests/metrics` directory. The JSON results files are placed into the `tests/metrics/results` directory.